cmd/wallet-js-sdk/src/index.js

```diff
@@ -28,4 +28,5 @@ export { DeviceLogin } from "./device/login";
 export { DeviceRegister } from "./device/register";
 export { Client as GNAPClient } from "./gnap/client";
 export { HTTPSigner } from "./httpsig/signer";
+export { authorize, callback } from "./oidc/issuance/openid4ci";
```

Comment:
I guess we need to update the header on top of this file

Reply:
Don't think so? It already has the avast copyright notice.